### PR TITLE
Fix index for address query

### PR DESCRIPTION
**Pull request by isocolsky**

### CODE REVIEW

**Comment by matiu** [comment]
Good catch!

File: lib/storage.js [hunk 1]
```diff
@@ -50,7 +50,8 @@ Storage.prototype._createIndexes = function() {
     id: 1,
   });
   this.db.collection(collections.ADDRESSES).createIndex({
-    walletId: 1
+    walletId: 1,
+    createdOn: 1,
```
Comment (dabura667) [inline lib/storage.js]: delete the comma ;-)

Comment (isocolsky) [inline lib/storage.js]: lol, I love to be able to just end with a comma, I don't have to change two lines if I need to add another field :smile:

Comment (dabura667) [inline lib/storage.js]: I wish. haha

**Pull request merged**